pkg/util/k8sutil/k8sutil.go

```diff
@@ -109,8 +109,8 @@ func makeRestoreInitContainerSpec(backupAddr, token, version string, m *etcdutil
 	return string(b)
 }
 
-func EtcdImageName(version string) string {
-	return fmt.Sprintf("quay.io/coreos/etcd:v%v", version)
+func EtcdImageName(baseImage, version string) string {
```

Comment:
It doesn't make sense to call this `EtcdImageName` anymore.
Can you change it to `imageName`?